app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -517,8 +517,8 @@ Talk to your provider about whether your experience makes you eligible for this
 
 Or, find other ways you can [train to be a teacher](/train-to-be-a-teacher).
 
-### If you're based overseas or are a non-UK citizen
+### If you're based outside the UK or are a non-UK citizen
```

Comment:
Might be clearer to say 'live outside the UK'? how do we define 'based'?

Reply:
```suggestion
### If you live outside the UK or are a non-UK citizen
```